disnake/ext/tasks/__init__.py

```diff
@@ -695,21 +731,55 @@ def change_interval(
                 self._handle.recalculate(self._next_iteration)
 
 
+T_co = TypeVar("T_co", covariant=True)
+L_co = TypeVar("L_co", bound=Loop, covariant=True)
+
+
+class Object(Protocol[T_co, P]):
+    def __new__(cls) -> T_co:
```

Comment:
```suggestion
L_co = TypeVar("L_co", bound=Loop, covariant=True)


class Object(Protocol[L_co, P]):
    def __new__(cls) -> L_co:
```

Reply:
It's made to be generic for anything to comply with the same implementation I made for `ui.button` & `ui.select`.
Ideally the `Object` would be moved from both here and `ui.item` to some kind of disnake typeshed, but that was out of the scope of either PR

Reply:
I see, fair enough.